### DISCUSSION TITLE
Provide ability to edit additional sensor options for Riemann-Sum & Utility Meter helpers

### DISCUSSION THREAD

**Discussion by danielbrunt57** [home-assistant/frontend#14276] [Configuration]
If possible, please enhance the UI for the following helpers to be able to modify additional parameters:

Riemann-Sum:
- Input sensor

Utility Meter:
- Meter reset offset

Nice to have:
Riemann-Sum:
- Integration method
- Metric prefix
- Time unit

Utility Meter:
- Supported tariffs
- Net consumption
- Delta values

**Comment by igorsantos07** [comment]
It's definitely weird to not being able to tweak these options, and even worse, double-check what you had entered when creating the helpers. That makes it hard (impossible?) to even debug why the helper is not working properly.

Another helper which is missing some editing options is #16568.

> This was already reported a little later, wrongly, at issue #14759.

**Comment by karwosts** [comment]
I believe this is not actually a frontend issue. Integrations need to implement an options flow to allow changing options after entity creation. 

The supported options flow is a feature of /core/, not the frontend. So this would need one core PR for each of those integrations to support additional configuration options.

**Reply by danielbrunt57** [reply]
> I believe this is not actually a frontend issue. Integrations need to implement an options flow to allow changing options after entity creation.
>
While that may be true, the issue as I see it is not a bug but a deficiency so one presumably cannot use github to post about it as that resource is for bug reporting.

> The supported options flow is a feature of /core/, not the frontend. So this would need one core PR for each of those integrations to support additional configuration options.
>
If I had the slightest clue of how to change it I would post the PR's...
